src/runtime-tools/linux/setup.sh

```diff
@@ -144,16 +144,18 @@ if type apt > /dev/null 2> /dev/null; then
     curl --retry 10 -sSL https://dot.net/v1/dotnet-install.sh -o dotnet-install.sh 2>&1 | logger -s -i -t 'onefuzz-curl-dotnet-install'
     chmod +x dotnet-install.sh
 
-    logger "running dotnet install"
-    /bin/bash ./dotnet-install.sh --version "$DOTNET_VERSION" --install-dir "$DOTNET_ROOT" 2>&1 | logger -s -i -t 'onefuzz-dotnet-setup'
+    for version in "${DOTNET_VERSIONS[@]}"; do
+        logger "running dotnet install $version"
+        /bin/bash ./dotnet-install.sh --version "$version" --install-dir "$DOTNET_ROOT" 2>&1 | logger -s -i -t 'onefuzz-dotnet-setup'
```

Comment:
What's the interaction here when we install 2 different versions to the same `install-dir`? For example, in line 156 (`"$DOTNET_ROOT"/dotnet tool install dotnet-dump --version 6.0.351802 --tool-path /onefuzz/tools 2>&1 | logger -s -i -t 'onefuzz-dotnet-tools'`), will it be using dotnet 6 or 7? Do we need to install the same tool twice but with each version of dotnet? How do we decide which version is being used?

Reply:
I think it will install it for the latest framework you have installed that the tool supports. In this case [`dotnet-dump` supports net6.0](https://www.nuget.org/packages/dotnet-dump#supportedframeworks-body-tab). 

Note that for these tools we don't need to have them on the same framework as our code, since we don't run any .NET on the nodes, and the tool does not attach to any running .NET code, it is used to analyze dumps after they are created.

Reply:
In the case you invoke it with `dotnet` it will use the framework that the executable is targetted at, so we have both v6 and v7 installed at the moment.